### ISSUE TITLE
Feat: Integrated with OpenCost for Cost Monitoring

### ISSUE THREAD

**Issue by elliotxx** [KusionStack/karpor#456]
### What would you like to be added?

Integrated with OpenCost for Cost Monitoring

### Why is this needed?

Integrated with OpenCost for Cost Monitoring

**Comment by cheny-alf** [comment]
I'm interested in the  issue. Is it still open for contributions? If so, I'd love to give it a try.

**Comment by elliotxx** [comment]
@cheny-alf  Ohh, you are a warrior, this issue is a bit difficult. If you are interested, I am willing to assign it to you. At present, it is just a vague idea, with uncertainty. We should start more discussions around it.

**Comment by cheny-alf** [comment]
@elliotxx  Of course, I'd love to. Before I get started, I should probably get a quick start with OpenCost. Also, could you give me a heads-up on what features we're aiming to implement for this issue?

**Comment by elliotxx** [comment]
@cheny-alf My preliminary idea is to view the cost monitor of all resources in the current range through OpenCost on an resource aggregate page of Karpor. For example, you can see the CPU, Memory cost (such as 1000¥) of all resources under the current namespace page.
I believe this is a very intuitive and valuable feature.

**Comment by elliotxx** [comment]
@cheny-alf Hi, did you encounter any problems during the implementation of this issue?

**Comment by cheny-alf** [comment]
@elliotxx Apologies for the late reply. I've been a bit swamped lately. I'm hoping to get back on track and start focusing on this issue. Looking forward to diving in. 
However, I encountered a problem when I tried to reinstall, and I've raised it in issue #649 . I hope you can take a look at it.
